### PR TITLE
Metadata for Episode 22 with Neil and Simon

### PR DESCRIPTION
**Pull request by pweschmidt**
The metadata for the forthcoming episode on Thursday

### CODE REVIEW

File: _posts/2020-06-11-richard-fitzjohn.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
```
Comment (vsoch) [inline _posts/2020-06-11-richard-fitzjohn.md]: Should this commit be here?

File: _posts/2020-06-25-huechong-hettrick.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: "Software is the backbone"
+author: "@pweschmidt"
+rse: "Neil Chue Hong and Simon Hettrick"
+phenotype: neilsimon-rse-phenotype.png
+excerpt: "For ca 10 years the Software Sustainability Institute in the UK has been the backbone of research enginnering. But how did they fare during the coronavirus pandemic and how will research software engineering change in future?"
```
Comment (vsoch) [inline _posts/2020-06-25-huechong-hettrick.md]: Did you mean to say “ca” ?

File: _posts/2020-06-25-huechong-hettrick.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+   url: https://twitter.com/SoftwareSaved 
+ - name: Neil Chue Hong
+   url: https://uk.linkedin.com/in/neilchuehong
+ - name: Simon Hettrick
+   url: https://uk.linkedin.com/in/simon-hettrick-2668a9a
+--- 
+
+In today's episode Peter meets Neil Chue Hong and Simon Hettrick. Neil and Simon are the director/deputy director of the Software Sustainability Institute (https://www.software.ac.uk). For ca 10 years the institute has been the "backbone" for research software engineering and the RSE movement. 
+It helped build the RSE movement and network, not only in the UK but in other countries. Neil and Simon discuss what changes the coronavirus pandemic brought to research software engineering. And what challenges and opportunities lie ahead of us. And did you know the Institut has a blog with cooking recipes?
+
```
Comment (vsoch) [inline _posts/2020-06-25-huechong-hettrick.md]: Institute

File: _posts/2020-06-25-huechong-hettrick.md (hunk 2 above)
Comment (vsoch) [reply _posts/2020-06-25-huechong-hettrick.md]: Looks like engineering is spelled wrong.

Comment (vsoch) [reply _posts/2020-06-25-huechong-hettrick.md]: “In the future”

File: _posts/2020-06-25-huechong-hettrick.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+ - name: Software Sustainability Institute on Twitter
+   url: https://twitter.com/SoftwareSaved 
+ - name: Neil Chue Hong
+   url: https://uk.linkedin.com/in/neilchuehong
+ - name: Simon Hettrick
+   url: https://uk.linkedin.com/in/simon-hettrick-2668a9a
+--- 
+
+In today's episode Peter meets Neil Chue Hong and Simon Hettrick. Neil and Simon are the director/deputy director of the Software Sustainability Institute (https://www.software.ac.uk). For ca 10 years the institute has been the "backbone" for research software engineering and the RSE movement. 
+It helped build the RSE movement and network, not only in the UK but in other countries. Neil and Simon discuss what changes the coronavirus pandemic brought to research software engineering. And what challenges and opportunities lie ahead of us. And did you know the Institut has a blog with cooking recipes?
```
Comment (vsoch) [inline _posts/2020-06-25-huechong-hettrick.md]: Probably you could do an and, and then continue the sentence (combine these two).

**Comment by vsoch** [comment]
Hey Peter sorry for the late review! If you don’t get around to it don’t worry I’ll fix the issues and make the post. All the content is there which is what is important :) I think I missed it this morning in a sea of SORSE update emails, sorry about that!

File: _posts/2020-06-11-richard-fitzjohn.md (hunk 1 above)
Comment (pweschmidt) [reply _posts/2020-06-11-richard-fitzjohn.md]: oh I don't know how that slipped in. That's the previous episode.

File: _posts/2020-06-25-huechong-hettrick.md (hunk 2 above)
Comment (pweschmidt) [reply _posts/2020-06-25-huechong-hettrick.md]: Thanks for spotting the typo in engineering. "Ca" is short for circa, will replace it with about. "In Future" actually works I think.

File: _posts/2020-06-25-huechong-hettrick.md (hunk 3 above)
Comment (pweschmidt) [reply _posts/2020-06-25-huechong-hettrick.md]: yep - will need to fix that

File: _posts/2020-06-25-huechong-hettrick.md (hunk 4 above)
Comment (pweschmidt) [reply _posts/2020-06-25-huechong-hettrick.md]: Done

**Comment by pweschmidt** [comment]
Hi Vanessa. Many thanks for casting an eye over this and my apologies for the mistakes. I just committed the changes to the file.

**Comment by vsoch** [comment]
It’s ok, I’ll try to cherry pick or just redo tomorrow. What happened is that your last pull request was squashed and merged, which means that the commits are not present in the master branch. In the future you should start with the current updated master branch commits and then check out a new branch from there, that way your old commits are no longer represented. Github is seeing your old commits and that they aren’t in the master branch and correctly showing that it’s new content, but of course that’s not what we want!

Anyhoo I’m off to bed!

**Comment by pweschmidt** [comment]
Oh - I see. Maybe we should discuss briefly how we do branching etc in future. At the moment I am doing this from the forked repo as discussed initially. Again, sorry for the extra work. 
Have a good night and talk to you tomorrow your time.
Best wishes, Peter

**Comment by vsoch** [comment]
You basically always want to keep your master branch in sync with the upstream master. This means that the commits that go into your feature branches likely won’t be in that master because we do a squash if those commits. So for your routine it just means that you check out master, pull updates from the upstream master, and then check out a new branch from it. I think you ran into issues here because you’ve only been using your master branch. this would work if you pulled from the upstream master and then over wrote some of your get history, but it’s actually a lot easier to just check out a separate feature branch that you can later throw away. For the next time that you do it before you start work we can talk through the commands on Slack. Cheers!

File: _posts/2020-06-25-huechong-hettrick.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+layout: post
+title: "Software is the backbone"
+author: "@pweschmidt"
+rse: "Neil Chue Hong and Simon Hettrick"
+phenotype: neilsimon-rse-phenotype.png
+excerpt: "For about 10 years the Software Sustainability Institute in the UK has been the backbone of research software engineering. But how did their work fare during the coronavirus pandemic? And how will research software engineering change in future?"
+date: 2020-06-11 8:30:00
+external_media: https://us-rse.org/rse-stories-episodes-1/2020/rse-stories-huechong-hettrick-episode-22.mp3
+length: 18840264
+duration: "00:39:16"
```
Comment (vsoch) [inline _posts/2020-06-25-huechong-hettrick.md]:
```suggestion
duration: "00:39:15"
```

Comment (vsoch) [reply _posts/2020-06-25-huechong-hettrick.md]: This is the time that the browser shows, so want to honor that.

**Review by vsoch — APPROVED** [review]
Good job on fixing up the files here! Week after next I'd be happy to walk through the git checkout steps with you so those other commits don't show up. I'll squash and merge here so hopefully it's not an issue.

**Pull request merged**